_includes/header.html

```diff
@@ -63,6 +63,7 @@
 			  <li role="separator" class="divider"></li>
 			  <li class="dropdown-header">Basics</li>
 			  <li><a href="{{ site.baseurl }}/contribute/contribution-guide/">Contribution Guide</a></li>
+			  <li><a href="{{ site.baseurl }}/contribute/ide-setup/">IDE setup</a></li>
```

Comment:
Not sure about a dedicated section for IDE setup. It could be part of the `Contribution Guide`. WDYT ?